vpd-manager/src/bios_handler.cpp

```diff
@@ -386,8 +437,22 @@ void IbmBiosHandler::saveCreateDefaultLparToVpd(
 
     if (auto l_pVal = std::get_if<types::BinaryVector>(&l_kwdValueVariant))
     {
+        commonUtility::toLower(
+            const_cast<std::string&>(i_createDefaultLparVal));
+
+        if (((((*l_pVal).at(0) & 0x02) == 0x02) &&
```

Comment:
Add comment to say that 2nd bit is being set here.
Todo to remove magic numbers